cli/compose/convert/compose_test.go

```diff
@@ -108,6 +108,77 @@ func TestNetworks(t *testing.T) {
 	assert.DeepEqual(t, []string{"special"}, externals)
 }
 
+func TestNetworksCreatedWithoutServices(t *testing.T) {
+	namespace := Namespace{name: "foo"}
+	serviceNetworks := map[string]struct{}{}
+	source := networkMap{
+		"normal": composetypes.NetworkConfig{
```

Comment:
I think only one NetworkConfig, whatever it is, is enough to test the feature? The test would be then more compact and less redundant.

Reply:
did you mean, add the content of the `TestNetworksCreatedWithoutServices` into the `TestNetworks`?

Reply:
No, I meant remove all network configs except one, like named, in this test 😄